network/topology/randomizedTopology.go

```diff
@@ -15,15 +17,19 @@ import (
 // By random topology we mean a node is connected to any other co-channel nodes with some
 // edge probability.
 type RandomizedTopology struct {
-	me      flow.Identifier             // used to keep identifier of the node
-	state   protocol.State              // used to keep a read only protocol state
-	subMngr network.SubscriptionManager // used to keep track topics the node subscribed to
-	chance  uint64                      // used to translate connectedness probability into a number in [0, 100]
-	rng     random.Rand                 // used as a stateful random number generator to sample edges
+	identifier flow.Identifier             // used to keep identifier of the node
```

Comment:
I think the existing name `me` (or `myID`/`myNodeID` which we use elsewhere) is a more descriptive name. It's more important to know what the identifier refers to than that it is an identifier (we already know that from the type!)

Reply:
Likewise for the other fields where this was changed

Reply:
applied